dev/tidb-in-kubernetes/initialize-cluster.md

```diff
@@ -35,7 +35,17 @@ category: how-to
 
     该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。
 
-2. 部署集群
+2. 设置允许访问 TiDB 的主机
+
+    在部署集群前可以通过设置 `permitHost` 设置允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
```

Comment:
```suggestion
    在部署集群前可以通过设置 `permitHost` 允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
```

Reply:
Suggest keeping the original description.